docs/assets/main.js

```
@@ -0,0 +1,2204 @@
+        (t.QueryParser.parseClause = function (e) {
+          var n = e.peekLexeme();
+          if (n != null)
+            switch (n.type) {
+              case t.QueryLexer.PRESENCE:
+                return t.QueryParser.parsePresence;
+              case t.QueryLexer.FIELD:
+                return t.QueryParser.parseField;
+              case t.QueryLexer.TERM:
+                return t.QueryParser.parseTerm;
+              default:
+                var r = 'expected either a field or a term, found ' + n.type;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)